kustomize/skipper/kustomization.yaml

```diff
@@ -0,0 +1,67 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base/prometheus/
+  - ../base/flagger/
+  - https://raw.githubusercontent.com/zalando/skipper/v0.11.140/docs/kubernetes/deploy/deployment/rbac.yaml
+  - https://raw.githubusercontent.com/zalando/skipper/v0.11.140/docs/kubernetes/deploy/deployment/service.yaml
+  - https://raw.githubusercontent.com/zalando/skipper/v0.11.140/docs/kubernetes/deploy/deployment/deployment.yaml
+  - namespace.yaml
+patchesStrategicMerge:
+  - patch.yaml
+patches:
+  - target:
+      kind: Deployment
+      name: skipper-ingress
+    patch: |-
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: skipper-ingress
+      spec:
+        template:
+          metadata:
+            annotations:
+              prometheus.io/path: /metrics
+              prometheus.io/port: "9911"
+              prometheus.io/scrape: "true"
+          spec:
+            nodeSelector:
+              $patch: delete
+            affinity:
+              $patch: delete
+            containers:
+              - name: skipper-ingress
+                image: registry.opensource.zalan.do/pathfinder/skipper:latest
```

Comment:
Please use a fixed version not latest.

Reply:
ok, was a result of request from Skipper devs, but as you prefer

Reply:
Maybe we should remove Skipper from the kustomize dir and tell people to install it from the official repo like we say in the tutorial. People would install Skipper from their official docs and then install Flagger and Prometheus in the `flagger-system`, feels strange to deploy the ingress in flagger-system namespace.

Reply:
Skipper is installed into `kube-system` as per default, not into `flagger-system` namespace.

The kustomize files here are necessary also for the `e2e` tests and provide some working setting for Skipper we require - like prometheus metrics.
Kustomize we worked out in order not to mirror the deployment but just change what we need.

To me it does not look different than other predefined setups of other mesh provider, or what am I missing?

Reply:
There is no kustomization in Flagger's repo that comes with an ingress or service mesh, you can imagine it's not possible to maintain such a thing. If you look at the other kustomization you'll see that it's for patching Flagger flags nothing more.

Reply:
It's unfortunate that Skipper goes to kube-system, but I see no issues with it from a Flagger perspective, you can deploy Flagger+Promethues in flagger-system and it should work file. Please move the Skipper kustomization in the test dir.

Reply:
fine with me!

Reply:
implemented accordingly